### PR TITLE
Fix Fedora based tests

### PR DESCRIPTION
**Pull request by saschagrunert**
Trying to find out why the CI is broken for Fedora.

### CODE REVIEW

File: test/helpers.bash [hunk 1]
```diff
@@ -38,7 +38,7 @@ ULIMITS=${ULIMITS:-}
 DEVICES=${DEVICES:-}
 OVERRIDE_OPTIONS=${OVERRIDE_OPTIONS:-}
 RUNTIME_PATH=$(command -v $CONTAINER_RUNTIME || true)
-RUNTIME_BINARY=${RUNTIME_PATH:-/usr/local/sbin/runc}
+RUNTIME_BINARY=${RUNTIME_PATH:-$(which runc)}
```
Comment (TomSweeneyRedHat) [inline test/helpers.bash]: interesting twist, I like!

**Comment by saschagrunert** [comment]
I think the issue is:
```
which: no runc in (/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin)
```

File: contrib/test/integration/main.yml [hunk 2]
```diff
@@ -89,6 +89,11 @@
     - name: clone build and install crun
       include: "build/crun.yml"
       when: "build_crun | default(False) | bool"
+
+    - name: clone build and install runc
+      include: "build/runc.yml"
+      when: "build_runc | default(True) | bool"
```
Comment (saschagrunert) [inline contrib/test/integration/main.yml]: @giuseppe Is it correct that `default(True)` only applies if `build_runc` is not set? That would mean in our case that it is always set becuse of vars.yml: `build_runc: False`.

**Comment by saschagrunert** [comment]
Look how green :green_heart: PTAL @haircommander @giuseppe @mrunalp @rhatdan

File: contrib/test/integration/main.yml (hunk 2 above)
Comment (giuseppe) [reply contrib/test/integration/main.yml]: I guess so :)  we can probably drop that

**Comment by giuseppe** [comment]
/test e2e_fedora

LGTM

File: contrib/test/integration/main.yml (hunk 2 above)
Comment (saschagrunert) [reply contrib/test/integration/main.yml]: Ah okay I'll do.

**Comment by saschagrunert** [comment]
One more round, I'm optimistic that this works :)

**Pull request merged**